cmd/init.go

```diff
@@ -271,5 +271,6 @@ func init() {
 	initCmd.PersistentFlags().StringArrayVar(&initOptions.OrgNames, "org-name", []string{}, "Organization name")
 	initCmd.PersistentFlags().StringArrayVar(&initOptions.NodeNames, "node-name", []string{}, "Node name")
 	initCmd.PersistentFlags().BoolVar(&initOptions.RemoteNodeDeploy, "remote-node-deploy", false, "Enable or disable deployment of FireFly contracts on remote nodes")
+	initCmd.PersistentFlags().StringToStringVar(&initOptions.EnvironmentVars, "environment-vars", map[string]string{}, "Common environment variables to set on all containers in FireFly stack")
```

Comment:
I wonder if we should keep it as a `map[string]interface{}` ?

Reply:
I think we should keep this as a string. Environment variables won't be nested, they are always in the format var_name=value. Also, they will always be in string format as well, so we should keep the type safety.